### PR TITLE
Task/reuse apply expression current context

### PR DESCRIPTION
**Pull request by AlvaroVega**
maybe related issue https://github.com/telefonicaid/iotagent-node-lib/issues/1305

### CODE REVIEW

**Comment by AlvaroVega** [comment]
> No CNR entry needed? No modifications to documentation?


Sure, CNR updated in https://github.com/telefonicaid/iotagent-node-lib/pull/1370/commits/1a1adf303db0dbe7936f9cf84d5e1809f9b4abde

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Add: save result of apply expression attribute into current context for futher usages
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: No associated issue?

(Fine with that, just checking in the case we are missing it... :)

**Comment by fgalan** [comment]
@mapedraza maybe something could be added to documention to task/reuse_apply_expression_current_context before merging this PR.

**Comment by fgalan** [comment]
> @mapedraza maybe something could be added to documention to task/reuse_apply_expression_current_context before merging this PR.

It will be done in separate PR.

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**